pipelines/base/config-dump.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: dump-config
+spec:
+  params:
+  - name: REPO_NAME
+    type: string
+    default: .github
+  - name: BRANCH_NAME
+    type: string
+    default: add-initial-peribolos-config
+  - name: BASE_BRANCH
+    type: string
+    default: main
```

Comment:
We shouldn't care about branch here I think. If you clone a repo, you are automatically checked to the default branch. Let's use the default branch right away.

Reply:
Discussed in a meeting.